scene/audio/audio_stream_player.cpp

```diff
@@ -286,6 +300,9 @@ StringName AudioStreamPlayer::get_bus() const {
 			return bus;
 		}
 	}
+
+	WARN_PRINT("There is no bus '" + bus + "' defined. Defaulting to 'Master'");
```

Comment:
One more thing actually: you should use `"` instead of `'`, to be consistent with other editor strings. Same in the other warning.